src/components/LastUpdatedText.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+
+export default class LastUpdatedText extends React.Component {
+
+  static propTypes = {
+    lastUpdatedString: React.PropTypes.string.isRequired,
+  };
+  constructor(props) {
+    super(props)
+    this.state = { timeDifferenceInMinutes: 0 }
```

Comment:
could you type the state ?

Reply:
In js?

Reply:
Could be the occasion to gradually add TS

Reply:
There are a LOT of tutorials online about how to migrate a JS codebase to TS (gradually)

Reply:
I would say that since we haven't done any of the initial typescript setup, (and since we have a ticket to do that), I'll save that for a future refactor